src/icesat2_tracks/local_modules/m_general_ph3.py

```diff
@@ -22,7 +22,8 @@
 from matplotlib import dates
 import datetime as DT
 #from cdo import *   # python version
-import m_tools_ph3 as MT
+#from Chess import <class-name>
+from . import m_tools_ph3 as MT
```

Comment:
You could make this an absolute import to be sure you're getting the right thing.
```suggestion
from icesat2_tracks.plot import m_tools_ph3 as MT
```

Reply:
It's a file that needs to import from another file in the same folder. I dont see why we should use the absolute import.

Reply:
Future discuss if we need to keep these relative paths. It might break in the future when auto test are included

Reply:
#32